docs/REFERENCE.md

```diff
@@ -708,18 +708,25 @@ To access the HTTP status code of the `default` response use `backend_responses.
 | `json_decode`                  | various         | Parses the given JSON string and, if it is valid, returns the value it represents.                                                                                                                                                                                                                   | `encoded` (string)                  | `json_decode("{\"foo\": 1}")`                        |
 | `json_encode`                  | string          | Returns a JSON serialization of the given value.                                                                                                                                                                                                                                                     | `val` (various)                     | `json_encode(request.context.myJWT)`                 |
 | `jwt_sign`                     | string          | jwt_sign creates and signs a JSON Web Token (JWT) from information from a referenced [JWT Signing Profile Block](#jwt-signing-profile-block) (or [JWT Block](#jwt-block) with `signing_ttl`) and additional claims provided as a function parameter.                                                                                                 | `label` (string), `claims` (object) | `jwt_sign("myJWT")`                                  |
+| `keys`                         | list            | Takes a map and returns a sorted list of the map keys.                                                                                                                                                                                                                                               | `inputMap` (object or map)          | `keys(request.headers)`                              |
+| `length`                       | integer         | Returns the number of elements in the given collection.                                                                                                                                                                                                                                              | `collection` (tuple, list or map; **no object**)   | `length([0,1,2,3])`                                  |
+| `lookup`                       | various         | Performs a dynamic lookup into a map. There are two required arguments, map and key, plus an optional default, which is a value to return if no key is found in map.                                                                                                                                 | `inputMap` (object or map), `key` (string), `default` (various) | `lookup({a = 1}, "b", "def")` |
```

Comment:
This is from the documentation on https://pkg.go.dev/github.com/zclconf/go-cty/cty/function/stdlib#Lookup. But when I use the `lookup()` function with only two parameters, I get an error.
Is this a version issue? Or an error in the stdlib documentation?

Reply:
@johakoch  It depends on the input value, the type check implementation seems to make a difference between object(attribute?) and maps/collection(index, key?) , then the 3rd argument is required. Maybe we should copy that stdlib and adapt this.

Reply:
I corrected the documentation.